tests/functional/disaster-recovery/regional-dr/test_show_replication_delays.py

```diff
@@ -0,0 +1,284 @@
+        )
+
+        scheduling_interval = dr_helpers.get_scheduling_interval(
+            rdr_workload[0].workload_namespace
+        )
+        wait_time = 2 * scheduling_interval  # Time in minutes
+        logger.info(f"Waiting for {wait_time} minutes to run IOs")
+        sleep(wait_time * 60)
+
+        primary_cluster_name = dr_helpers.get_current_primary_cluster_name(
+            rdr_workload[0].workload_namespace
+        )
+        secondary_cluster_name = dr_helpers.get_current_secondary_cluster_name(
+            rdr_workload[0].workload_namespace, rdr_workload[0].workload_type
+        )
+
+        acm_obj = AcmAddClusters()
+        page_nav = ValidationUI()
+        page_nav.refresh_web_console()
+
+        config.switch_to_cluster_by_name(primary_cluster_name)
+        before_failover_last_group_sync_time = []
+        for obj in drpc_objs:
+            before_failover_last_group_sync_time.append(
+                dr_helpers.verify_last_group_sync_time(obj, scheduling_interval)
+            )
+        logger.info("Verified lastGroupSyncTime in CLI")
+
+        logger.info("Verifying the DR health status on UI")
+        check_dr_status(
+            acm_obj, workload_names, rdr_workload, expected_status="healthy"
+        )
+
+        # Bring DOWN the corresponding deployment
+        if pvc_interface == constants.CEPHBLOCKPOOL:
+            logger.info("Change replica count to 0 for rbd-mirror on the secondary ")
+            modify_rbd_replica_count(replica_count=0)
+        else:
+            logger.info("Change replica count to 0 for mds deployment on the primary ")
+            modify_mds_replica_count(replica_count=0)
+
+        # Wait for the range of interval between 2x - 3x interval to
+        # validate the message "warning"
+        # on the UI under Application -> DR Status
+        logger.info(
+            "Waiting for interval between the sync interval time and 2x of sync interval"
+            " to validate 'warning' state"
+        )
+        wait_time = scheduling_interval + 2
+        sleep(wait_time * 60)
+
+        check_dr_status(
+            acm_obj, workload_names, rdr_workload, expected_status="warning"
+        )
+
+        logger.info("Waiting to validate 'critical' state")
+        wait_time = 3 * scheduling_interval
+        sleep(wait_time * 60)
+
+        check_dr_status(
+            acm_obj, workload_names, rdr_workload, expected_status="critical"
+        )
+
+        # Bring UP the corresponding deployment
+        if pvc_interface == constants.CEPHBLOCKPOOL:
+            logger.info("Change replica count to 1 for rbd-mirror on the secondary ")
+            modify_rbd_replica_count(replica_count=1)
+        else:
+            logger.info("Change replica count to 1 for mds deployment on the primary ")
+            modify_mds_replica_count(replica_count=1)
+
+        logger.info(
+            "Waiting for the first sync to happen after scaling up the deployment"
```

Comment:
Can you add a check for lastGroupSyncTime?

Reply:
We are validating dr health status from UI, which operates based on lastGroupSyncTime in drpc.
Do we still need lastGroupSyncTime check explicitly?